src/Tests/RemoteUnitTestExecutor.Host_x86/RemoteUnitTestExecutor.Host_x86.csproj

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I will add another host project for x64 to avoid loading x64 assembly into 32-bit processes.